scheduler/task_priority_cmp.go

```diff
@@ -148,7 +147,7 @@ func (c *byTaskGroupOrder) compare(t1, t2 task.Task, _ *CmpBasedTaskComparator)
 	}
 
 	// If tasks are in the same task group and build, apply the task group comparator.
-	reason = "tasks are in same group, higher task is earlier"
+	reason = "earlier in sequence"
```

Comment:
It's not immediately obvious to me that "sequence" refers to a task group. Maybe "same group, earlier in sequence, so ranked first"?

Reply:
changed to "earlier in the same task group"